worker-workflow/src/main/resources/workflow-control.js

```diff
@@ -184,6 +184,16 @@ function fieldExists(document, fieldName) {
     return document.getField(fieldName).hasValues();
 }
 
+function isNotEmptyMap(obj) {   
```

Comment:
not a great parameter name, obj is actually expected to be a jsonString

Reply:
Confusing negation.  Wouldn't an `isEmptyMap()` helper function make more sense?
(Also some whitespace at the end of the line)